deploy/k8s/charts/trustification/templates/helpers/_image.tpl

```diff
@@ -35,7 +35,8 @@ Arguments (dict):
   * image - image object
 */}}
 {{- define "trustification.common.imageName" }}
-{{- with .image.fullName }}
+{{- $fullName := .image.fullName | default .defaults.fullName }}
+{{- with $fullName }}
 {{- . }}
 {{- else }}
 {{- include "trustification.common.imageRegistry" . }}/{{ .image.name | default .imageName | default .defaults.name | default "trust" }}:{{ include "trustification.common.imageVersion" . }}
```

Comment:
@ctron Should we add the conditional block for the SHA value check to `trustification.common.imageVersion`? This might help us when the use has sha value on `image.version` field.

```suggestion
  {{- $imageVersion := include "trustification.common.imageVersion" . }}
  {{- if eq (substr 0 7 $imageVersion) "sha256:" }}
    {{- include "trustification.common.imageRegistry" . }}/{{ .image.name | default .imageName | default .defaults.name | default "trust" }}@{{ $imageVersion }}
  {{- else }}
    {{- include "trustification.common.imageRegistry" . }}/{{ .image.name | default .imageName | default .defaults.name | default "trust" }}:{{ $imageVersion }}
  {{- end }}
```

Reply:
I think the `fullName` field already is the condition. Which can be used for other use cases as well. I don't see a benefit of adding a third option.